core/meterer/meterer.go

```diff
@@ -60,10 +52,15 @@ func (m *Meterer) Start(ctx context.Context) {
 		ticker := time.NewTicker(1 * time.Hour)
 		defer ticker.Stop()
 
+		// initial tick immediately upto Start
```

Comment:
maybe we can put this outside the goroutine and immediately fail & return error if this initial sync fails?

Reply:
makes sense, I moved it out:)